packages/utils/test/validators.spec.ts

```diff
@@ -106,4 +106,167 @@ describe("Validators", () => {
       }),
     ).to.be.false;
   });
+  it("should validate namespaces v1", () => {
```

Comment:
Would avoid using `v1/2/3` terminology here since it could imply we have different versioning for namespaces

```suggestion
  it("should validate namespaces (configuration 1)", () => {
```